sites/mosul/configs/openfn/adaptor_registry_cache.json

```diff
@@ -0,0 +1 @@
{"version":"0.4.7"}],"repo":"git+https://github.com/openfn/adaptors.git","latest":"0.4.7"},{"name":"@openfn/language-kobotoolbox","versions":[{"version":"1.0.4"},{"version":"1.1.0"},{"version":"1.1.1"},{"version":"1.2.0"},{"version":"1.3.0"},{"version":"1.3.1"},{"version":"1.3.2"},{"version":"1.3.3"},{"version":"2.0.1"},{"version":"2.1.0"},{"version":"2.2.0"},{"version":"2.3.0"},{"version":"2.4.0"},{"version":"2.4.1"},{"version":"2.4.2"},{"version":"2.4.3"}],"repo":"git+https://github.com/openfn/adaptors.git","latest":"2.4.3"},{"name":"@openfn/language-azure-storage","versions":[{"version":"1.0.0"},{"version":"1.0.1"},{"version":"1.0.2"},{"version":"1.1.0"},{"version":"2.0.0"},{"version":"2.0.1"},{"version":"2.0.2"},{"version":"2.0.3"},{"version":"2.0.4"},{"version":"2.0.5"}],"repo":"git+https://github.com/openfn/adaptors.git","latest":"2.0.5"},{"name":"@openfn/language-nexmo","versions":[{"version":"0.2.0"},{"version":"0.2.1"},{"version":"0.2.2"},{"version":"0.2.3"},{"version":"0.3.1"},{"version":"0.4.0"},{"version":"0.4.1"},{"version":"0.4.2"},{"version":"0.4.3"},{"version":"0.4.4"},{"version":"0.4.5"},{"version":"0.4.6"},{"version":"0.4.7"},{"version":"0.4.8"},{"version":"0.4.9"},{"version":"0.5.0"},{"version":"0.5.1"},{"version":"0.5.2"},{"version":"0.5.3"},{"version":"0.5.4"},{"version":"0.5.5"},{"version":"0.5.6"},{"version":"0.5.7"},{"version":"0.5.8"},{"version":"0.5.9"}],"repo":"git+https://github.com/openfn/adaptors.git","latest":"0.5.9"},{"name":"@openfn/language-mailgun","versions":[{"version":"0.2.2"},{"version":"0.3.0"},{"version":"0.3.1"},{"version":"0.3.2"},{"version":"0.3.3"},{"version":"0.3.4"},{"version":"0.4.1"},{"version":"0.4.2"},{"version":"0.4.3"},{"version":"0.4.4"},{"version":"0.4.5"},{"version":"0.5.0"},{"version":"0.5.1"},{"version":"0.5.2"},{"version":"0.5.3"},{"version":"0.5.4"},{"version":"0.5.5"},{"version":"0.5.6"}],"repo":"git+https://github.com/openfn/adaptors.git","latest":"0.5.6"},{"name":"@openfn/language-openlmis","versions":[{"version":"1.0.0"},{"version":"1.0.1"},{"version":"1.0.2"},{"version":"1.0.3"},{"version":"1.0.4"},{"version":"1.0.5"},{"version":"1.0.6"},{"version":"1.0.7"}],"repo":"git+https://github.com/openfn/adaptors.git","latest":"1.0.7"},{"name":"@openfn/language-fhir","versions":[{"version":"1.1.0"},{"version":"1.1.1"},{"version":"1.1.2"},{"version":"1.1.3"},{"version":"1.1.4"},{"version":"1.1.5"},{"version":"2.0.0"},{"version":"3.0.1"},{"version":"3.1.0"},{"version":"3.1.1"},{"version":"3.1.2"},{"version":"3.2.0"},{"version":"4.0.0"},{"version":"5.0.0"},{"version":"5.0.1"},{"version":"5.0.2"},{"version":"5.0.3"}],"repo":"git+https://github.com/openfn/adaptors.git","latest":"5.0.3"},{"name":"@openfn/language-test-tmp","versions":[{"version":"1.0.0-next.1"},{"version":"1.0.0-next.2"},{"version":"1.0.0-next.4"},{"version":"1.0.0-next.5"},{"version":"1.0.0-next.6"},{"version":"1.0.0-next.9"},{"version":"1.0.1-next.0"}],"repo":"git+https://github.com/openfn/adaptors.git","latest":"1.0.1-next.0"}]
```

Comment:
Shouldn't this cache _only_ include the adaptors you've installed [here](https://github.com/MSF-OCG/LIME-EMR/pull/82/files#diff-b8c170569f12e25717fd3a3be71d17f36ebdc82d1754411ab49ae1d76c2e23dcR37) ?

We don't want them selecting adaptors that are not already installed locally, as there will be no internet access at runtime.

Reply:
Hmm, yeah, so that will stop the user from making an obvious error 🤔 .

I think it's fine to have all the adaptors, because then if they want to use a new adaptor, all they have to do is run the command `openfn repo install -a <adaptor@version>`.

If we filter out that list, then it means they will have to generate that file every single time as well.

Reply:
Yeah exactly, that's the tradeoff. I _think_ the short list is better, but let's see which they prefer!